lib/components/UploadPicker.vue

```diff
@@ -306,7 +306,8 @@ export default defineComponent({
 			return this.queue?.filter((upload: Upload) => upload.status === UploadStatus.FAILED).length !== 0
 		},
 		isUploading(): boolean {
-			return this.queue?.length > 0
+			// also ignore cancelled uploads
+			return this.queue?.filter((upload: Upload) => upload.status !== UploadStatus.CANCELLED).length !== 0
```

Comment:
```suggestion
			return this.queue?.filter((upload: Upload) => upload.status !== UploadStatus.CANCELLED).length > 0
```

Or am I wrong?
- queue undefined -> `false`
- queue only cancelled -> `false`
- queue has some non cancelled -> `true`

Because `undefined !== 0` is true so with your change it would report uploading in that case (but `undefined > 0` is false)

Reply:
you're right! absolutely